.docker/Dockerfile

```diff
@@ -27,25 +27,25 @@ RUN wget http://ftp.acc.umu.se/pub/GNOME/sources/aravis/0.6/aravis-0.6.4.tar.xz
     checkinstall --pkgname aravis --requires="libglib2.0-dev, libxml2-dev" && \
     ldconfig
 
-RUN wget https://software.micro-epsilon.com/scanCONTROL-Linux-SDK-0-2-4.zip && \
-    unzip scanCONTROL-Linux-SDK-0-2-4.zip && \
-    rm scanCONTROL-Linux-SDK-0-2-4.zip && \
-    cd /scanCONTROL\ Linux\ SDK\ 0.2.4/libmescan/ && \
+RUN wget https://software.micro-epsilon.com/scanCONTROL-Linux-SDK-0-2-5.zip && \
```

Comment:
You could use an `ARG` and set it to a default value (ie: the version you're targetting) to keep this DRY.

Reply:
Done :)